[v1.1.3] Release Staking Fixes (#239)
polkadot-fellows/runtimes: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

## [1.1.3] 15.03.2024

### Fixed

> - Bumps staking pallet version which applies a patch to prevent staking controllers from becoming stashes which may lead to ledger inconsistencies. Patch and context: [paritytech/polkadot-sdk#3639](https://github.com/paritytech/polkadot-sdk/pull/3639).

Reviewer: ```suggestion
- Bumps Staking pallet version. This patch prevents staking controllers from becoming stashes, which may lead to ledger inconsistencies. Patch and context: [paritytech/polkadot-sdk#3639](https://github.com/paritytech/polkadot-sdk/pull/3639).
```
